src/cls/rgw/cls_rgw.cc

```diff
@@ -1759,6 +1759,11 @@ static int rgw_bucket_link_olh(cls_method_context_t hctx, bufferlist *in, buffer
     if (ret < 0) {
       return ret;
     }
+
+    // if the OLH not exist, error out
```

Comment:
Maybe add "does" as in "if the OLH does not exist, error out".

Reply:
Ok, thanks for pointing out